src/static.jl

```diff
@@ -3,66 +3,39 @@ using Distributed: @spawn
 using Base.Iterators: flatten
```

Comment:
I believe `using Distributed: @spawn` is no longer needed and we can remove its dependency.

Reply:
True.